tests/integration/debug/test_debug.py

```diff
@@ -13,23 +13,23 @@ def schema(self):
     def models(self):
         return "models"
 
-    def run_and_test(self, contains_catalog: bool):
+    def run_and_test(self):
```

Comment:
These tests changed because I now explicitly set the catalog to hive_metastore if it is set to None (after checking the cluster setting).  We were defaulting to spark_catalog, but I couldn't find where this was set, and it was causing me pain.

Reply:
So question: is there any situation where changing this default behavior is breaking?